packages/app/src/pages/WelcomePage/WalletPage.test.tsx

```diff
@@ -0,0 +1,55 @@
+import "cross-fetch/polyfill";
+import {
+  fireEvent,
+  renderWithRouter,
+  screen,
+  waitFor,
+} from "@fuels-ui/test-utils";
+
+import App from "~/App";
+
+export const checkFirstLoad = async () => {
+  await renderWithRouter(<App />, { route: "/pool/list" });
```

Comment:
why do you need to be in `/pool/list` route if you're testing Wallet-related stuff?

Reply:
this test is to check if we access any route, user will be redirected to `CreateWallet` page instead. I'm using `/pool/list`, but could by any other route.